compliance/admin.py

```diff
@@ -17,11 +18,21 @@ class ExportsInquiryLogAdmin(admin.ModelAdmin):
 
     model = ExportsInquiryLog
     search_fields = ["user__email", "computed_result", "info_code", "reason_code"]
-    list_filter = ["computed_result", "info_code", "reason_code"]
-    list_display = ["user", "computed_result", "info_code", "reason_code"]
+    list_filter = [
+        "computed_result",
+        "info_code",
+        "reason_code",
+        "user__legal_address__country",
+    ]
+    list_display = ["user", "computed_result", "info_code", "reason_code", "country"]
+    list_select_related = ["user__legal_address"]
     readonly_fields = [] if settings.DEBUG else get_field_names(ExportsInquiryLog)
     actions = ["manually_approve_inquiry"]
 
+    def country(self, instance):
+        """ generate country name from pycountry """
+        return pycountry.countries.get(alpha_2=instance.user.legal_address.country).name
```

Comment:
If a country's abbreviation doesn't match any country in pycountry, it will give an error. I think we should do some exception handling here.

Reply:
@HamzaIbnFarooq that's not going to happen because it's coming from the user profile legal_address which is already validating that. 

Note: We are not putting anything manually this is what ever we are getting from the user profile data which is already validated.

https://github.com/mitodl/bootcamp-ecommerce/blob/2eea9ef8a62dd9c6b992598893072f2d179676d7/profiles/models.py#L108-L110

https://github.com/mitodl/bootcamp-ecommerce/blob/2eea9ef8a62dd9c6b992598893072f2d179676d7/profiles/models.py#L57-L71